### PR TITLE
6280: Compare solutions document

### PR DESCRIPTION
**Pull request by talawson05**
created docker compose and pipeline yaml for file uploader
[AB#6280](https://buyingcatalog.visualstudio.com/c5f97979-5b03-4d10-ba8d-871d0526b408/_workitems/edit/6280)

### CODE REVIEW

File: azure-pipelines-file-uploader.yml [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+  displayName: 'Run DAPI Integration tests'
+  inputs:
+    command: test
+    projects: '**/*Integration.Tests/*.csproj'
+    arguments: '-v n'
+
+
+- script: docker-compose -f "docker-compose.integration.yml" down -v
+  displayName: 'Docker-compose down'
+  condition: always()
```
Comment (adrian-fitsios) [inline azure-pipelines-file-uploader.yml]: I'm not quite sure that this is necessary / useful.

DAPIs integration tests do not use the file-uploader in any shape or form, so running them isn't going to test anything.

I'd suggest removing this part. What do you think?

**Review by adrian-fitsios — CHANGES_REQUESTED** [review]

File: azure-pipelines-file-uploader.yml (hunk 1 above)
Comment (talawson05) [reply azure-pipelines-file-uploader.yml]: oh, i thought the integration tests used those documents. if not then I'll pull them out

Comment (adrian-fitsios) [reply azure-pipelines-file-uploader.yml]: The integration tests are independent and set up using [this](https://github.com/nhs-digital-gp-it-futures/BuyingCatalogueDocumentService/blob/3439c98e676529045e8f252f8ac3c5c51e6cd6e1/test/NHSD.BuyingCatalogue.Documents.API.Integration.Tests/Steps/AzureBlobSteps.cs#L19)
where they can insert data from the 'SampleData' directory into the  blob store container.

**Review by adrian-fitsios — DISMISSED** [review]
👍 Good work as always

**Review by adrian-fitsios — APPROVED** [review]
Good as a quickfix to the problem, not as a proper solution, we might need to revisit this in the future.

**Pull request merged**